config/db_tools.standalone.sample.yaml

```
@@ -9,104 +9,82 @@
 
 # Working directory is the path to which all relative file references will
 # be relative to. If none set, the path will be this file directory instead.
-workdir: /var/www/my_project/
+# In most cases, you will not need it.
+#workdir: /var/www/my_project/
 
-# Database connections.
-# One line per connection, a single database URL, all options as query
-# parameters. Connections will be made using makincorpus/query-builder
-# which will raise exceptions when invalid options are found.
-# There is less configuration amplitude than using doctrine/dbal in
-# Symfony, yet it should be enough in most case.
-# In case any options or specific behaviour is missing, please file
-# an issue at https://github.com/makinacorpus/php-query-builder/issues
-connections:
-    connection_one: "pgsql://username:password@hostname:port?version=16.0&other_option=..."
-    connection_two: "mysql://username:password@hostname:port?version=8.1&other_option=..."
+# Where to put generated backups.
+# Root directory of the backup storage manager. Default filename
+# strategy will always use this folder as root path.
+storage_directory: ./var/db_tools
```

if this is the default one, should it be commented ?

```suggestion
#storage_directory: ./var/db_tools
```

---

Yes, indeed.